### PR TITLE
Add tombstone validation

### PR DESCRIPTION
**Pull request by felixfontein**
Here we go. Please use `Merge by Rebase` (or squash, doesn't matter for one commit) to avoid another merge commit :)

### CODE REVIEW

File: test/lib/ansible_test/_data/sanity/code-smell/runtime-metadata.py [hunk 1]
```diff
@@ -72,9 +72,10 @@ def main():
 
     plugin_routing_schema = Any(
         Schema({
-            ('deprecation'): Any(deprecation_schema),
+            ('deprecation'): Any(deprecation_tombstoning_schema),
+            ('tombstone'): Any(deprecation_tombstoning_schema),
```
Comment (felixfontein) [inline test/lib/ansible_test/_data/sanity/code-smell/runtime-metadata.py:93]: I guess it doesn't make sense to have both at the same time, but let's sort this out later. It's more important to get a basic validation.

**Comment by felixfontein** [comment]
Oh, wait. I need to add one more commit (for YAML date support).

**Comment by felixfontein** [comment]
@gundalow I'm done. This should do what we want (and decided that today's meeting) :)

CC @nitzmahone

**Comment by gundalow** [comment]
Thanks for all your support with this, merged.

**Pull request merged**